sdk/scheduler/src/main/java/com/mesosphere/sdk/state/StateStoreUtils.java

```diff
@@ -304,4 +310,45 @@ public static void setLastCompletedUpdateType(
             return ConfigurationUpdater.UpdateResult.DeploymentType.valueOf(value);
         }
     }
+
```

Comment:
Looks like these methods are only used in StateResource. Can just move them to private methods in StateResource.